apps/frontend/src/components/global/groups/Users.vue

```diff
@@ -173,15 +173,22 @@ export default class Users extends Vue {
     this.editedUserID = '0';
   }
 
-  deleteUserConfirm(): void {
+  @Emit()
+  deleteUserConfirm(): boolean {
+    let saveable = true;
+    const userToDelete = this.currentUsers.indexOf(this.getEditedUser());
+    if (
+      this.currentUsers[userToDelete].groupRole === 'owner' &&
+      this.numberOfOwners() <= 1
+    ) {
+      saveable = false;
+    }
     if (this.editedUserID !== '0') {
```

Comment:
why did we choose '0' as the default id?  isn't it possible for the default admin to be an edited user, and doesn't the default admin have an id of 0?

Reply:
No, the default admin has an id of 1.